### DISCUSSION TITLE
Conan Recipes

### DISCUSSION THREAD

**Discussion by whgibbo** [aws/aws-sdk-cpp#2903] [General]
Hi,
I see that somebody contributed conan recipe https://conan.io/center/recipes/aws-sdk-cpp for 1.9.234.  Which is a few years old..

Was curious if there are any plans to include conan 1.x / 2.x recipes ?

Thanks

**Comment by jmklix** [comment]
This is not something that we currently maintain. It would need to be updated [here](https://github.com/conan-io/conan-center-index/tree/master/recipes/aws-sdk-cpp), but that is not something that we currently plan on updating/supporting.
